website/src/styles/Chakra.tsx

```diff
@@ -1,24 +1,22 @@
-import { ChakraProvider, cookieStorageManagerSSR, localStorageManager } from "@chakra-ui/react";
-
+import { ChakraProvider } from "@chakra-ui/react";
 import { theme } from "./Theme";
 
 export function Chakra({ cookies, children }) {
-  const colorModeManager = typeof cookies === "string" ? cookieStorageManagerSSR(cookies) : localStorageManager;
+  const cookieColorValue = cookies["chakra-ui-color-mode"] ?? "light";
```

Comment:
it seems to me that it should be possible to use `document.cookie` as a parameter instead of having re-do everything ourselves.

Reply:
yeah.

Reply:
I'll try doing that.